<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HealBone-GaitAnalysis</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="KinectIRFOV">
       <attribute name="title">
        <string>IR FOV</string>
       </attribute>
       <widget class="QGraphicsView" name="cameraIrFovView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>740</width>
          <height>515</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="KinectFOV">
       <attribute name="title">
        <string>FOV</string>
       </attribute>
       <widget class="QGraphicsView" name="cameraFovView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>740</width>
          <height>515</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="KinectIR">
       <attribute name="title">
        <string>IR</string>
       </attribute>
       <widget class="QGraphicsView" name="cameraIrView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>740</width>
          <height>515</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="resultsDock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>258</width>
     <height>143</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>524287</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Gait Control Panel</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="resultsDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="btnStart">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>23</y>
          <width>111</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>开始步态检测</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sbTime">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>23</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>5</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>检测时长/sec</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbFPS">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>78</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>FPS_5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FPS_15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FPS_30</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>FPS</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DepthMode</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbDepthMode">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>78</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>NFOV_2X2BD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NFOV_UNBD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WFOV_2X2BD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WFOV_UNBD</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>113</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ColorResolution</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbColorResolution">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>132</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>RES_720P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RES_1080P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RES_1440P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RES_1536P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RES_2160P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RES_3072P</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>112</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ModelComplexity</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbModelComplexity">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>LOW</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MIDDLE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HIGH</string>
         </property>
        </item>
       </widget>
       <widget class="QSlider" name="hsMinDetectionConfidence">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="sliderPosition">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>219</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Min Detection Confidence</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>169</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Min Tracking Confidence</string>
        </property>
       </widget>
       <widget class="QSlider" name="hsMinTrackingConfidence">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="sliderPosition">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QCheckBox" name="cbSmoothLandmarks">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Smooth Landmarks</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="messagesDock">
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>101</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Analysis Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="messagesDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTextEdit" name="outputText">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="viewerDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>RealTime Human Skeleton Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="viewerDockContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QVTKWidget" name="vtkContext" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>322</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMaximumDockWidget" name="angleViewerDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>RealTime Angles Viewer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="angleViewerDockContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="angleViewerDockContentsVerticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="angleViewerDockScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="angleViewerDockScrollAreaContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>320</width>
          <height>356</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="angleViewerDockScrollAreaContentsLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1328</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>widgets.QVtkWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QMaximumDockWidget</class>
   <extends>QDockWidget</extends>
   <header>widgets.QMaximumDockWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>activateBrowserItem(QModelIndex)</slot>
  <slot>selectBrowserItem(QModelIndex)</slot>
  <slot>tabCloseRequested(int)</slot>
 </slots>
</ui>
